packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -3171,6 +3171,11 @@ class SemanticsConfiguration {
   }
 
   /// Whether the owning [RenderObject] is selected (true) or not (false).
+  ///
+  /// This is different from having accessibility focus.  The element that is
```

Comment:
Nit: one space after period
```suggestion
  /// This is different from having accessibility focus. The element that is
```